### PR TITLE
Nextjs switch

### PR DESCRIPTION
**Pull request by fuzzyers**
I have merged the back and frontend into a single fullstack NEXTJS

I have updated the instructions for setting up the API route in the [backend](https://github.com/OtagoPolytechnic/Cloudy-with-a-Chance-of-LoRa/wiki/Backend#backend-next)

### CODE REVIEW

File: fullstack/src/app/api/recieve-data/route.js
Comment (SamanthaCra) [inline fullstack/src/app/api/recieve-data/route.js:1]: Will communicate the schema to you this week to ensure this works with the Postgres database.

**Review by SamanthaCra — CHANGES_REQUESTED** [review]
Great job, dev runs well and everything is still nice and responsive.  I will have to sit down with you to ensure the build is working as I received an error. We use the build on the app.

Also the favicon and title are gone, they were in the app Ben built. 

Build error:
```Build error occurred
Error: Failed to collect page data for /api/recieve-data
    at D:\skool\Cloudy-with-a-Chance-of-LoRa\fullstack\node_modules\next\dist\build\utils.js:1268:15
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  type: 'Error'
}````

**Comment by fuzzyers** [comment]
> Great job, dev runs well and everything is still nice and responsive. We want to ensure the build is working as I received an error. We use the build on the application server.
> 
> Also the favicon and title are gone, they were in the app Ben built.
> 
> Build error:
> 
> ```
> Error: Failed to collect page data for /api/recieve-data
>     at D:\skool\Cloudy-with-a-Chance-of-LoRa\fullstack\node_modules\next\dist\build\utils.js:1268:15
>     at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
>   type: 'Error'
> }````
> ```


So the build Error seems to be related to the missing .env file and a database link as it cant compile the API so this wont be an issue in the proper build as we will have the .env in it I have also updated the backend wiki to talk about the setup process of this.

The favicon and title have been readded

**Review by SamanthaCra — APPROVED** [review]
Thanks for updating that and letting me know how the error can be fixed. I will look into updating Docker today.

**Comment by SamanthaCra** [comment]
You should be able to delete that Dockerfile if it's creating conflicts.

**Review by BenCampbellNelson — APPROVED** [review]
Great idea switching us over to Next.js. This decision is going to be great for the scalability of the project for the future and makes it a lot easier for routing and using the API.

**Pull request merged**